charts/oet/templates/oet.yaml

```diff
@@ -4,80 +4,51 @@
 apiVersion: v1
 kind: Service
 metadata:
-  name: oet-{{ template "oet.name" . }}-{{ .Release.Name }}
-  namespace: {{ .Release.Namespace }}
-  labels:
-    app: oet-{{ template "oet.name" . }}-{{ .Release.Name }}
-    chart: {{ template "oet.chart" . }}
-    release: {{ .Release.Name }}
-    heritage: {{ .Release.Service }}
+ name: oet-ssh
+ labels:
+  run: ssh
 spec:
-  type: ClusterIP
-  ports:
-  - name: oet-ssh
-    port: 22
-    protocol: TCP
-  selector:
-    app: oet-{{ template "oet.name" . }}-{{ .Release.Name }}
+ type: NodePort
+ ports:
+ - port: 2022
+   targetPort: 22
+   protocol: TCP
+ selector:
+  run: ssh
 
 ---
-apiVersion: apps/v1
-kind: StatefulSet
+apiVersion: v1
+kind: ConfigMap
 metadata:
-  name: oet-{{ template "oet.name" . }}-{{ .Release.Name }}
-  namespace: {{ .Release.Namespace }}
-  labels:
-    app: oet-{{ template "oet.name" . }}-{{ .Release.Name }}
-    chart: {{ template "oet.chart" . }}
-    release: {{ .Release.Name }}
-    heritage: {{ .Release.Service }}
+  name: "environment-oet-{{ template "oet.name" . }}-{{ .Release.Name }}"
+data:
+   environment: |
+     TANGO_HOST=databaseds-tango-base-{{ .Release.Name }}:10000
+---
+apiVersion: v1
+kind: Pod
+metadata:
+ name: oet-ssh
+ labels:
+  run: ssh
 spec:
-  selector:
-    matchLabels:
-      app: oet-{{ template "oet.name" . }}-{{ .Release.Name }}
-  serviceName: oet-{{ template "oet.name" . }}-{{ .Release.Name }}
-  replicas: 1
-  template:
-    metadata:
-      labels:
-        app: oet-{{ template "oet.name" . }}-{{ .Release.Name }}
-        chart: {{ template "oet.chart" . }}
-        release: {{ .Release.Name }}
-        heritage: {{ .Release.Service }}
-    spec:
-      initContainers:
-      - name: waitdatabaseds
-        image: nexus.engageska-portugal.pt/ska-telescope/oet:latest
-        imagePullPolicy: IfNotPresent
-        command:
-          - /usr/local/bin/wait-for-it.sh
-          - databaseds-tango-base-{{ .Release.Name }}:10000
-          - --timeout=30
-          - --strict
-          - --
-          - /venv/bin/itango3
-          - --profile=ska
-
-      containers:
-      - name: oet
-        image: "{{ .Values.oet.image.registry }}/{{ .Values.oet.image.image }}:{{ .Values.oet.image.tag }}"
-        imagePullPolicy: {{ .Values.oet.image.pullPolicy }}
-        ports:
-        - name: oet-ssh
-          containerPort: 22
-          protocol: TCP
-        resources:
-{{- with .Values.nodeSelector }}
```

Comment:
even if we don't use it at the moment, it could be useful for the future...  can you put it back?